index.html

```diff
@@ -75,24 +75,146 @@ <h2>QUESTION:</h2>
           adipisicing elit.
         </p>
         <ul role="list" class="flex flex-wrap gap-5 justify-evenly mt-20">
-          <li class="p__tag m-auto">HTML5</li>
-          <li class="p__tag m-auto">CSS3</li>
-          <li class="p__tag m-auto">Git</li>
-          <li class="p__tag m-auto">Pro</li>
+          <li class="card__tag m-auto">HTML5</li>
+          <li class="card__tag m-auto">CSS3</li>
+          <li class="card__tag m-auto">Git</li>
+          <li class="card__tag m-auto">Pro</li>
         </ul>
       </article>
     </main>
+    <main
```

Comment:
It seems like you already combined your htmls? Very impressive! 
But as far as I know, you can only have one `<main>` per html page. You could try `<section>` instead of `<main>` and only use one `<main>` to wrap it all up. :)

Reply:
i think you're right. I will fix this - thx

Reply:
Jerry just said, that you can also use `<main>`. I didn´t know that :D. I guess you can do it either way.